editor/plugins/material_editor_plugin.cpp

```diff
@@ -268,7 +293,8 @@ MaterialEditor::MaterialEditor() {
 	viewport->add_child(camera);
 
 	light1 = memnew(DirectionalLight3D);
-	light1->set_transform(Transform3D().looking_at(Vector3(-1, -1, -1), Vector3(0, 1, 0)));
+	light1->set_transform(Transform3D().looking_at(Vector3(1, -1, -1), Vector3(0, 1, 0)));
+	light1->set_shadow(true);
```

Comment:
The preview environment scene is very small in size, so we can use a single shadow cascade to improve performance and reduce draw calls:

```suggestion
	light1->set_shadow(true);
	light1->set_shadow_mode(DirectionalLight3D::SHADOW_ORTHOGONAL);
```